.github/dependabot.yml

```diff
@@ -1,7 +1,27 @@
+# Docs: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
 version: 2
 updates:
+- package-ecosystem: gradle
+  directory: "/data-prepper-api"
+  schedule:
+    interval: daily
```

Comment:
QUES: Could daily update be too frequent. Especially for API and core library.

Reply:
Ah good point - I think we can change it to "weekly" once the initial round of dependency upgrades are complete. I'll create a follow-up PR later.